### PR TITLE
Add new keyboard Promise87

### PR DESCRIPTION
**Pull request by spbgzh**
## Description

add new keyboard promise87
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/wuque/promise87/ansi/rules.mk [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = yes           # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no       # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
+
+LAYOUT = 87_ansi
```
Comment (zvecr) [inline keyboards/wuque/promise87/ansi/rules.mk:20]:
```suggestion
```

File: keyboards/wuque/promise87/wkl/rules.mk [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = yes           # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no       # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
+
+LAYOUT = 87_wkl
```
Comment (zvecr) [inline keyboards/wuque/promise87/wkl/rules.mk:20]:
```suggestion
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/wuque/promise87/ansi/anis.c [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "anis.h"
```
Comment (fauxpark) [inline keyboards/wuque/promise87/ansi/anis.c:17]:
```suggestion
#include "ansi.h"
```
Typo here and in the filename.

File: keyboards/wuque/promise87/ansi/readme.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+* Keyboard Maintainer: [spbgzh](https://github.com/spbgzh)
+* Hardware Supported: promise 87
+* Hardware Availability: [promise87](https://shop.wuquestudio.com/)  
+​
+
+Make example for this keyboard (after setting up your build environment):  
+
+    make wuque/promise87/ansi:default
+Flashing example for this keyboard:
```
Comment (fauxpark) [inline keyboards/wuque/promise87/ansi/readme.md:14]:
```suggestion

Flashing example for this keyboard:
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/wuque/promise87/ansi/anis.c (hunk 3 above)
Comment (spbgzh) [reply keyboards/wuque/promise87/ansi/anis.c]: deleted 'anis.c' and added 'ansi.c'

File: keyboards/wuque/promise87/ansi/readme.md (hunk 4 above)
Comment (spbgzh) [reply keyboards/wuque/promise87/ansi/readme.md]: solved

File: keyboards/wuque/promise87/wkl/rules.mk (hunk 2 above)
Comment (spbgzh) [reply keyboards/wuque/promise87/wkl/rules.mk]: deleted

File: keyboards/wuque/promise87/ansi/rules.mk (hunk 1 above)
Comment (spbgzh) [reply keyboards/wuque/promise87/ansi/rules.mk]: deleted

**Review by fauxpark — APPROVED** [review]

File: keyboards/wuque/promise87/wkl/rules.mk (hunk 2 above)
Comment (spbgzh) [reply keyboards/wuque/promise87/wkl/rules.mk]: deleted

File: keyboards/wuque/promise87/ansi/rules.mk (hunk 1 above)
Comment (spbgzh) [reply keyboards/wuque/promise87/ansi/rules.mk]: deleted

**Pull request merged**